python/ray/autoscaler/_private/resource_demand_scheduler.py

```diff
@@ -308,3 +382,48 @@ def _inplace_subtract(node: ResourceDict, resources: ResourceDict) -> None:
         assert k in node, (k, node)
         node[k] -= v
         assert node[k] >= 0.0, (node, k, v)
+
+
+def _inplace_add(a: Dict, b: Dict) -> None:
+    for k, v in b.items():
+        a[k] += v
+
+
+def placement_groups_to_resource_demands(placement_group_load: List[
+        PlacementGroupTableData]):
+    """Preprocess placement group requests into regular resource demand vectors
+    when possible.
+
+    Args:
+        placement_group_load (List[PlacementGroupData]): List of
+        PlacementGroupLoad's.
+
+    Returns:
+        (resource_demand_vector: List[ResourceDict], unconverted:
+        List[List[ResourceDict]]) Converts PACK, SPREAD, and STRICT_PACK
```

Comment:
Can you split this into two seperate returns and not put them in a Tuple?

Reply:
Two return values is a tuple...

Reply:
You seem to add a description of the functionality in the returns "Converts PACK...."

Reply:
I meant:
resource_demand_vector (List[ResourceDict]):  XXXXX The updated resource demand vector...
unconverted (List[List[ResourceDict]]): YYY The remaining unconverted....